src/app/components/app-job/app-job.component.html

```diff
@@ -74,12 +74,17 @@ <h1 class="bold">
                     </p>
                   </div>
                   <div class="column is-2">
-                    <p class="total-time">
-                      <span class="icon">
-                        <img src="images/icons/clock.svg">
-                      </span>
-                      <span>{{ job?.time }}</span>
-                    </p>
+                    <div class="column is-12">
+                      <p class="total-time">
+                        <span class="icon">
+                          <img src="images/icons/clock.svg">
+                        </span>
+                        <span>{{ job?.time }}</span>
+                      </p>
+                    </div>
+                    <div class="column is-12" *ngIf="job.status === 'running' && job.lastJob">
+                      <progress class="progress is-small" value="{{expectedProgress}}" max="1">{{expectedProgress}}%</progress>
```

Comment:
you can bind to value like this: `[value]="expectedProgress"`, without parenthesis.